### ISSUE TITLE
Getting assert errors on regression tests

### ISSUE THREAD

**Issue by goelakash** [weecology/retriever#448]
When running the `nosetests` command, I get Assertion errors in test_regresson_cli.py, specifically for:

- PostgreSQLRegression
- DownloadRegression (for the DelMoral dataset)

I am not sure about the `known_md5s` values here.
(Ubuntu 15.10, MySQL - 5.6, PostgreSQL - 9.4.6)

**Comment by henrykironde** [comment]
@goelakash  what version of MysQL and PostgreS are you using 
can you share the details of the error

**Comment by goelakash** [comment]
Here's the complete log:

```
Downloading scripts...
Download Progress: [#############################################] 100.00%
The retriever is up-to-date
rm: cannot remove ‘output_file*’: No such file or directory
=> Installing Avian Body Size Ecological Archives 2007
Inserting rows to output_file_species: 3769 / 3769
Done!
.=> Installing Vegetation plots - del Moral, 2010
Inserting rows to output_file_species_plot_year: 148155 / 148155
Inserting rows to output_file_species: 85 / 85
Inserting rows to output_file_structure_plot_year: 1743 / 1743
Inserting rows to output_file_plots: 96 / 96
Done!
.=> Installing Masses of Mammals (Ecological Archives 2003)
Inserting rows to output_file_MOM: 5730 / 5730
Done!
.=> Installing Avian Body Size Ecological Archives 2007
Copying avian_ssd_jan07.txt from raw_data/AvianBodySize
Couldn't copy file to raw_data/AvianBodySize/
Done!
.=> Installing Vegetation plots - del Moral, 2010
Copying MSH_SPECIES_PLOT_YEAR.csv from raw_data/DelMoral2010
Couldn't copy file to raw_data/DelMoral2010/
Copying MSH_STRUCTURE_PLOT_YEAR.csv from raw_data/DelMoral2010
Couldn't copy file to raw_data/DelMoral2010/
Copying MSH_PLOT_DESCRIPTORS.csv from raw_data/DelMoral2010
Couldn't copy file to raw_data/DelMoral2010/
Copying MSH_SPECIES_DESCRIPTORS.csv from raw_data/DelMoral2010
Couldn't copy file to raw_data/DelMoral2010/
Done!
F=> Installing Masses of Mammals (Ecological Archives 2003)
Copying MOMv3.3.txt from raw_data/MoM2003
Couldn't copy file to raw_data/MoM2003/
Done!
.=> Installing Avian Body Size Ecological Archives 2007
Creating database testdb...
Creating table testdb.species...
Inserting rows to testdb.species: 3769 / 3769
Done!
.=> Installing Vegetation plots - del Moral, 2010
Creating database testdb...
Creating table testdb.species_plot_year...
Inserting rows to testdb.species_plot_year: 148155 / 148155
Creating table testdb.species...
Inserting rows to testdb.species: 85 / 85
Creating table testdb.structure_plot_year...
Inserting rows to testdb.structure_plot_year: 1743 / 1743
Creating table testdb.plots...
Inserting rows to testdb.plots: 96 / 96
Done!
.=> Installing Masses of Mammals (Ecological Archives 2003)
Creating database testdb...
Creating table testdb.MOM...
Inserting rows to testdb.MOM: 5730 / 5730
Done!
.NOTICE:  drop cascades to table testschema.mom
DROP SCHEMA
=> Installing Avian Body Size Ecological Archives 2007
Creating database testschema...
Creating table testschema.species...
Inserting rows to testschema.species: 3769 / 3769
Done!
FNOTICE:  drop cascades to table testschema.species
DROP SCHEMA
=> Installing Vegetation plots - del Moral, 2010
Creating database testschema...
Creating table testschema.species_plot_year...
Inserting rows to testschema.species_plot_year: 148155 / 148155
Creating table testschema.species...
Inserting rows to testschema.species: 85 / 85
Creating table testschema.structure_plot_year...
Inserting rows to testschema.structure_plot_year: 1743 / 1743
Creating table testschema.plots...
Inserting rows to testschema.plots: 96 / 96
Done!
FNOTICE:  drop cascades to 4 other objects
DETAIL:  drop cascades to table testschema.species_plot_year
drop cascades to table testschema.species
drop cascades to table testschema.structure_plot_year
drop cascades to table testschema.plots
DROP SCHEMA
=> Installing Masses of Mammals (Ecological Archives 2003)
Creating database testschema...
Creating table testschema.MOM...
Inserting rows to testschema.MOM: 5730 / 5730
Done!
Frm: cannot remove ‘output_database’: No such file or directory
=> Installing Avian Body Size Ecological Archives 2007
Creating table AvianBodySize_species...
Inserting rows to AvianBodySize_species: 3769 / 3769
Done!
.=> Installing Vegetation plots - del Moral, 2010
Creating table DelMoral2010_species_plot_year...
Inserting rows to DelMoral2010_species_plot_year: 148155 / 148155
Creating table DelMoral2010_species...
Inserting rows to DelMoral2010_species: 85 / 85
Creating table DelMoral2010_structure_plot_year...
Inserting rows to DelMoral2010_structure_plot_year: 1743 / 1743
Creating table DelMoral2010_plots...
Inserting rows to DelMoral2010_plots: 96 / 96
Done!
.=> Installing Masses of Mammals (Ecological Archives 2003)
Creating table MoM2003_MOM...
Inserting rows to MoM2003_MOM: 5730 / 5730
Done!
........................
======================================================================
FAIL: test_DelMoral2010 (test_regression_cli.DownloadRegression)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/goelakash/dev/retriever/test/test_regression_cli.py", line 82, in <lambda>
    stub_test = lambda self: getattr(self, test_method)(*args)
  File "/home/goelakash/dev/retriever/test/test_regression_cli.py", line 139, in check_download_regression
    assert current_md5 == known_md5
AssertionError

======================================================================
FAIL: test_AvianBodySize (test_regression_cli.PostgreSQLRegression)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/goelakash/dev/retriever/test/test_regression_cli.py", line 82, in <lambda>
    stub_test = lambda self: getattr(self, test_method)(*args)
  File "/home/goelakash/dev/retriever/test/test_regression_cli.py", line 130, in check_postgres_regression
    assert current_md5 == known_md5
AssertionError

======================================================================
FAIL: test_DelMoral2010 (test_regression_cli.PostgreSQLRegression)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/goelakash/dev/retriever/test/test_regression_cli.py", line 82, in <lambda>
    stub_test = lambda self: getattr(self, test_method)(*args)
  File "/home/goelakash/dev/retriever/test/test_regression_cli.py", line 130, in check_postgres_regression
    assert current_md5 == known_md5
AssertionError

======================================================================
FAIL: test_MoM2003 (test_regression_cli.PostgreSQLRegression)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/goelakash/dev/retriever/test/test_regression_cli.py", line 82, in <lambda>
    stub_test = lambda self: getattr(self, test_method)(*args)
  File "/home/goelakash/dev/retriever/test/test_regression_cli.py", line 130, in check_postgres_regression
    assert current_md5 == known_md5
AssertionError

----------------------------------------------------------------------
Ran 38 tests in 107.295s

FAILED (failures=4)
```

**Comment by henrykironde** [comment]
ok for postgres use 9.1
then I guess we shall be left with only DelMoral2010 fails

**Comment by ethanwhite** [comment]
DelMoral2010 fixed by a7b7beefd8f1fb30c0597977c89b45fb9ec14411.

**Comment by ethanwhite** [comment]
Thanks for reporting this @goelakash. That was definitely a weird one.
